script/build.js

```diff
@@ -14,7 +14,7 @@ const permalinks = require('metalsmith-permalinks');
 const sitemap = require('metalsmith-sitemap');
 const watch = require('metalsmith-watch');
 
-const webpackMetalsmithConnect = require('../config/webpack-metalsmith-connect');
+const webpackMetalsmithConnect = require('../vagov-apps/config/webpack-metalsmith-connect');
```

Comment:
By "modularizing Webpack from Metalsmith", what we mean is that the entire app-code/Webpack compilation is abstracted away from Metalsmith into this single import. The entire Webpack compilation and app-code can easily exist in its own repo, as shown here. `vagov-content` is the top-level repo, with a clone of `vets-website` installed into a directory `vagov-apps` as a submodule. We can now delete the `src` and `config` directories from `vagov-content`. Inversely, `vagov-apps` can safely remove the content directories, build script, etc. This is an example of how splitting the `vets-website` repo could work.

Reply:
We could also do this inversely, by setting up a repo for `vagov-apps` and converting `vets-website` into `vagov-content`.